DALI.tex

```diff
@@ -1003,29 +1009,43 @@ \subsection{UUID}
 datatype="char" arraysize="*" xtype="uuid"
 \end{verbatim}
 
-UUID values \citep{std:RFC4122} are serialised using the canonical ascii (hex) 
+UUID values \citep{std:RFC4122} are serialised using the canonical ascii (hex)
 representation, for example: e0b895ca-2ee4-4f0f-b595-cbd83be40b04.
 
+\subsection{JSON}
+
+For complex data blobs that are not intended to be queriable, publishers
+may choose to store them as serialised JSON \citep{std:JSON} strings in
+VOTables.  Note that in general, an appropriate VOTable serialisation
+(e.g., as a \xmlel{GROUP} for a \xmlel{PARAM} or as unrolled
+\xmlel{FIELD}\/s or a separate table for table cells) is preferred.
+
+\xmlel{PARAM}\/s and \xmlel{FIELD}\/s containing JSON values should be
```

Comment:
This could probably be simplified and restated in the style I ended up using for the other xtypes. maybe:
```
JSON values serialised in VOTable or described in service parameters should have the following VOTable type metadata:

\begin{verbatim}
datatype="char" arraysize="*" xtype="uri"
\end{verbatim}
```

The general allowance for unicodeChar and more specific arraysize would apply as it does elsewhere.